modules/visualizing-external-entities-known-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/visualizing-external-entities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="visualizing-external-entities-known-limitations_{context}"]
+= Known limitations
+
+[role="_abstract"]
+The following are some known limitations of the Visualizing external entities feature:
+
+* When you enable external IP collection for a cluster, Collector in those clusters reports more information to Sensor and to Central. It might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. Red{nbsp}Hat recommends that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
```

Comment:
```suggestion
* When you enable external IP collection for a cluster, Collector in those clusters report more information to Sensor and to Central. This might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. It is recommended that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
```